### PR TITLE
Support SPV_INTEL_complex_float_mul_div extension

### PR DESCRIPTION
**Pull request by KornevNikita**
Spec: https://github.com/intel/llvm/pull/5824
Intrinsics: https://reviews.llvm.org/D119284

### CODE REVIEW

**Comment by KornevNikita** [comment]
Waiting for llvm patch to merge.

**Comment by KornevNikita** [comment]
@MrSidims @vmaksimo @AlexeySachkov @mlychkov take a look please

File: lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h [hunk 1]
```diff
@@ -592,6 +592,8 @@ template <> inline void SPIRVMap<Capability, std::string>::init() {
       "GlobalVariableDecorationsINTEL");
   add(internal::CapabilityNonConstantAddrspacePrintfINTEL,
       "NonConstantAddrspacePrintfINTEL");
+  add(internal::CapabilityComplexFloatMulDivINTEL,
+      "ComplexFloatMulDivINTEL"); // INTEL
```
Comment (mlychkov) [inline lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h:596]:
```suggestion
      "ComplexFloatMulDivINTEL");
```

**Review by mlychkov — APPROVED** [review]
Other than one comment LGTM.

File: lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h (hunk 1 above)
Comment (KornevNikita) [reply lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h]: Oops. some type. Thanks [06a6048](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1442/commits/06a6048af20a92df8626f152afc316603d5f8c08)

**Review by MrSidims — APPROVED** [review]

**Pull request merged**